aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/execute_script.move

```diff
@@ -0,0 +1,15 @@
+//# init --addresses Alice=0xf75daa73fc071f93593335eb9033da804777eb94491650dd3f095ce6f778acb6 Bob=0x9c3b634ac05d0af393e0f93b9b19b61e7cac1c519f566276aa0c6fd15dac12aa
+//#      --private-keys Alice=56a26140eb233750cd14fb168c3eb4bd0782b099cde626ec8aff7f3cceb6364f Bob=952aaf3a98a27903dd078d76fc9e411740d2ae9dd9ecb87b96c7cd6b791ffc69
+//#      --initial-coins 10000
+
+//# run --script --signers Alice  --args @Bob 100
+script {
+    use AptosFramework::Coin;
+    use AptosFramework::TestCoin::TestCoin;
+
+    fun main(sender: &signer, receiver: address, amount: u64) {
+        Coin::transfer<TestCoin>(sender, receiver, amount);
```

Comment:
update casing

Reply:
Coin and TestCoin cases stayed the same after https://github.com/aptos-labs/aptos-core/pull/2018

Reply:
#2032 changes this and lands soon.